ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/LedgerClientWithoutLedgerId.scala

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.client
+
+import com.daml.grpc.adapter.ExecutionSequencerFactory
+import com.daml.ledger.api.v1.active_contracts_service.ActiveContractsServiceGrpc
+import com.daml.ledger.api.v1.admin.package_management_service.PackageManagementServiceGrpc
+import com.daml.ledger.api.v1.admin.party_management_service.PartyManagementServiceGrpc
+import com.daml.ledger.api.v1.command_completion_service.CommandCompletionServiceGrpc
+import com.daml.ledger.api.v1.command_service.CommandServiceGrpc
+import com.daml.ledger.api.v1.command_submission_service.CommandSubmissionServiceGrpc
+import com.daml.ledger.api.v1.package_service.PackageServiceGrpc
+import com.daml.ledger.api.v1.transaction_service.TransactionServiceGrpc
+import com.daml.ledger.api.v1.version_service.VersionServiceGrpc
+import com.daml.ledger.client.configuration.LedgerClientConfiguration
+import com.daml.ledger.client.services.acs.ActiveContractSetClientWithoutLedgerId
+import com.daml.ledger.client.services.admin.{PackageManagementClient, PartyManagementClient}
+import com.daml.ledger.client.services.commands.{
+  CommandClientWithoutLedgerId,
+  SynchronousCommandClient,
+}
+import com.daml.ledger.client.services.pkg.PackageClientWithoutLedgerId
+import com.daml.ledger.client.services.transactions.TransactionClientWithoutLedgerId
+import com.daml.ledger.client.services.version.VersionClientWithoutLedgerId
+import io.grpc.{Channel, ManagedChannel}
+import io.grpc.netty.NettyChannelBuilder
+
+import java.io.Closeable
+import java.util.concurrent.TimeUnit
+import scala.concurrent.ExecutionContext
+
+class LedgerClientWithoutLedgerId private (
```

Comment:
Use a package instead of a different name to distinguish from the with-ledger-ID variants. e.g. `com.daml.ledger.client.withoutledgerid`.